### DISCUSSION TITLE
Flutter and Postgres database

### DISCUSSION THREAD

**Discussion by shanxn** [community/community#146564] [Programming Help]
### Select Topic Area

Question

### Body

Hey I am working on a new project with postgresql I need to dev an app with flutter but postgresql as database I don’t know where to start I’m looking for someone way to do that any idea and suggestions will be appreciated. I have only worked with firebase and flutter!

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @DHARSHANKONO , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
